src/pages/Dashboard/NavBar/MonthNavigation.tsx

```diff
@@ -1,18 +1,26 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { ArrowIosBack, ArrowIosForward } from '@styled-icons/evaicons-solid';
 import ArrowButton from './ArrowButton';
 import lodash from 'lodash';
```

Comment:
I didn't saw that, but let's use 
```suggestion
import capitalize from 'lodash/capitalize';
```